nfd-daemonset-all-combined.yaml.template

```diff
@@ -1,137 +0,0 @@
-# This template contains an example of running nfd-master and nfd-worker in the
```

Comment:
Ah, This is a mistake from me here as we don't have nfd-worker in this manifest. Also the comment on the top of the manifest needs to be updated to This template contains an example of running nfd-master, nfd-worker and nfd-topology-updater in the same pod

Reply:
I think we should keep this manifest template

Reply:
done